cryptoki/src/object.rs

```diff
@@ -405,7 +413,6 @@ impl TryFrom<CK_ATTRIBUTE_TYPE> for AttributeType {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-#[non_exhaustive]
```

Comment:
Why this got removed? Correct me if I'm wrong but vendor defined doesn't cover all space of possible values? 🤔

Reply:
Sorry, fixed that. You're correct that it's not exhaustive.